### ISSUE TITLE
Don't explain SQL in mysql cheat

### ISSUE THREAD

**Issue by federico-razzoli** [cheat/cheatsheets#186]
`mysql` refers to the command line client that comes with [MySQL](https://dev.mysql.com/doc/refman/8.0/en/mysql.html) or [MariaDB](https://mariadb.com/kb/en/mysql-command-line-client/).

This cheat shouldn't show SQL examples. `psql` cheat doesn't, and I find it much cleaner.

**Comment by chrisallenlane** [comment]
Hi, @federico-razzoli 

Thanks for the suggestion.

I have mixed feelings about this. On one hand, you're clearly right - the SQL explanations are inconsistent with other cheatsheets, and cheatsheets generally are focused upon individual command-line applications.

On the other hand, (to the best of my knowledge), the SQL presented here is MySQL-specifc, and seems (to me) like it could be genuinely useful to some people. (I agree that "generic" SQL has no place here.)

That being said - how would you propose that we present the MySQL-specific SQL? In a `mysql-sql` cheatsheet or something, perhaps?

Thanks.

**Comment by federico-razzoli** [comment]
Yes, some of the statements are MySQL and MariaDB specific. However they're relevant no matter which client you use: mycli, MySQL Shell, mariadb, or a GUI.

A `mysql-sql` cheat would makes sense to me. The cheats for `mysql` and other clients for MySQL/MariaDB could end with a note like: "See mysql-sql for MySQL SQL dialect".

I'd be happy to contribute `mycli` and `mariadb` cheats, as well as other utilities.

**Comment by chrisallenlane** [comment]
> A `mysql-sql` cheat would makes sense to me.

I like this idea as well.

> I'd be happy to contribute mycli and mariadb cheats, as well as other utilities.

If you have the time/interest to make these changes, I would very much welcome the PR :+1:
